src/overlays/actors/ovl_En_Gm/z_en_gm.c

```diff
@@ -958,16 +1039,16 @@ s32 func_8094F904(EnGm* this, GlobalContext* globalCtx, struct_80133038_arg2* ar
     return ret;
 }
 
-s32 func_8094FAC4(EnGm* this, GlobalContext* globalCtx, struct_80133038_arg2* arg2) {
-    u16 sp2E = gSaveContext.time - 0x3FFC;
+s32 func_8094FAC4(EnGm* this, GlobalContext* globalCtx, ScheduleResult* arg2) {
+    u16 sp2E = SCHEDULE_TIME_NOW;
```

Comment:
So with this macro, could you explain what the `- 0x3FFC` was all about?

Reply:
The time range for in-game time started at a different point (0 is midnight) than scripts use (0 is dawn). The `0x3FFC` was the constant used to convert between the two, which in the macro is represented `0x10000 / 360 * 90`. `0x10000`, a full game day, multiplied to get 1/4, or 6 hours, to convert midnight to dawn (but, due to truncation, is slightly off. I wonder if that deserves a `//@ bug`?)